api/grpcserver/mesh_service.go

```diff
@@ -328,106 +266,67 @@ func (s MeshService) readLayer(
 	ctx context.Context,
 	layerID types.LayerID,
 	layerStatus pb.Layer_LayerStatus,
-) (*pb.Layer, error) {
-	// Load all block data
-	var blocks []*pb.Block
-
-	// Save activations too
-	var activations []types.ATXID
+) (pbLayer *pb.Layer, err error) {
```

Comment:
NIT: The named returns are not actually used (every `return` statement explicitly states what values are returned). If the names here are removed you can use `err` instead of `err2` everywhere in this function 🙂

Reply:
yea, oof, i like named returns so I added this, but our strict linting rules no longer allow this =\